hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacility.java

```diff
@@ -112,7 +116,7 @@ public void handleTxBody(@NonNull final HederaState state, @NonNull final Transa
     }
 
     @NonNull
-    private static Bytes getFileContent(@NonNull final HederaState state, @NonNull final FileID fileID) {
+    public static Bytes getFileContent(@NonNull final HederaState state, @NonNull final FileID fileID) {
```

Comment:
Why was this changed to public?

Reply:
The only reason I did that is to use it in the `SystemFileUpdateFacilityTest.throttleMangerUpdatedOnFileUpdate` test. If that's not a good enough argument I can make it private again and copy the method into the test class.

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏Perhaps make it package private and use `@VisibleForTesting`?